### DISCUSSION TITLE
Image component + lazy + no JavaScript

### DISCUSSION THREAD

**Discussion by aralroca** [vercel/next.js#22048] [Ideas]
**Describe the feature you'd like to request**

When the Image component is loaded with `loading="lazy"` the image is not rendered when the JavaScript is disabled. I imagine it's because the lazy control is through JavaScript instead of using the native `loading="lazy"` that modern browsers have. I propose to add the `noscript` tag as a fallback.

**Describe the solution you'd like**

It would be great to add the `noscript` tag with the `img` with the native `loading=lazy` as a fallback for those who browse without JavaScript.

```js
<noscript><img loading="lazy" alt="..." src="..." /></noscript>
```

**Describe alternatives you've considered**

An alternative would be to always use the native `loading=lazy` and add some polyfill for non-modern browsers.

**Comment by Kyoss79** [comment]
The implementation with <noscript> makes Chrome load all images on pageload. It does (on my machine) not respect the noscript tag and load all images anyway.

**Reply by Kyoss79** [reply]
It seems I only have this behaviour on production, not in dev.

**Comment by alanshortis** [comment]
Not overiding the native `loading` attribute unless the browser doesn't have support would be a good approach.

`next/image` not working without JS, and the native lazy loading not being an option is a dealbreaker for me. While I use Next for the developer experience, I don't send the JS bundle to users. I'd rather not send upwards of 400kb of additional JS to show images.
